### DISCUSSION TITLE
Circle images on player.

### DISCUSSION THREAD

**Discussion by Bitwolfies** [jellyfin/jellyfin-vue#2029] [UI & UX]
### Description of the bug

Not sure if its intended, but playback for some items are circular, as opposed to square.  (If it is intended, its not great looking as it's square everywhere else, nor have I ever seen an app that rounds out album art)
![image](https://github.com/jellyfin/jellyfin-vue/assets/65789901/c92dd82c-3f71-4d1a-a1f9-1628ea9ed075)

### Steps to reproduce

Play song
Its round

### Expected behavior

Be Square

### Platform

Linux

### Browser

Firefox

### Jellyfin server version

10.8.10

### Additional context
If intentional, I really quite hope you reconsider, the rounded corners in the standard album view look good, and should be used in the now playing. Aspect ratio should not be changed like this, it does not look good and honestly is enough to have me re consider using the client. Doesn't even make much sense as clicking the image brings you to the fullscreen view with proper square images.

**Comment by ferferga** [comment]
Hello @Bitwolfies, sorry for the delay in my response, but I'm overwhelmed by exams right now.

This started happening since the first moment I had the music player + Vuetify 3 (with md3 blueprint) + Vue 3 running in the Vue 3 switch. It's how avatars are now in MD3's specs, so Vuetify defaulted to those instead of the square ones.

As with the rest of the elements, I wanted to make a 1:1 match to the variant we had in then-master Vue 2. However, other things started taking priority and I ended up really liking it. It's true most music apps use square design for music albums, however:

* It's cool to be different
* I saw some cool designs where the music album art is resembled as a CD (maybe we can add a hole at some point too?). 
[1](https://dribbble.com/shots/10380614-Music-Player-Concept), [2](https://www.pinterest.es/pin/ui--396105729730302083/), [3](https://www.behance.net/gallery/120056815/Daily-UI-009-Music-player), [4](https://www.uplabs.com/posts/ui-design-minimal-music-player-concept)
Continuing this point... If you want to enjoy the album art, there's the fullscreen music UI already. I think showing the current track as a *CD* that is inside the deck makes sense somehow?

Any additional feedback from on this @ThibaultNocchi?

I'm turing this into a discussion as well, as it's more fitting now.

**Comment by Bitwolfies** [comment]
With all due respect to this being free software and not my project and ect:

"It's cool to be different" is quite frankly an *awful* justification for this, there's a reason literally no other music app does this. Jellyfin could crop all video to be a hexagon, it would certainly be different, but you don't cause it changes the intended display medium too much. Album art is square, it should be displayed as square, for the few that look ok cropped, dozens more look awful. 

An actual example of cool to be different was when JF displayed the covers of movies and shows in it's pause menu, that was actually cool. If there's any feature vue should add its that.

-album art makes for terrible cd covers, jamming a hole into the center of the album art is gonna make it look worse. Jellyfin actually supports disc art as a media source, if you wanted to do the cd idea you should start there.

-"Fullscreen ui has it" isn't really a justification, just cause one part of the client looks proper doesn't really justify another looking bad, it's also super inconsistent to click the circular image and receive the proper square one. If the rounded images supposedly look as good as is claimed, why aren't they used client wide? Why doesn't the library use them? 

Throughout all this I'm really not trying to be rude or ungrateful, I'm no ui designer or programmer, but I do genuinely think it looks bad, with the justifications being flimsy at best. Square art/rounded corners doesn't need justification, it's quite literally how 99% of albums are expected to look. It's not cool or necessary to change it.

That's just my opinion anyway.

**Comment by ferferga** [comment]
This is fixed in https://github.com/jellyfin/jellyfin-vue/pull/2301
